diff --git a/node_modules/react-native/Libraries/LogBox/UI/LogBoxInspector.js b/node_modules/react-native/Libraries/LogBox/UI/LogBoxInspector.js
index 110e411..9014139 100644
--- a/node_modules/react-native/Libraries/LogBox/UI/LogBoxInspector.js
+++ b/node_modules/react-native/Libraries/LogBox/UI/LogBoxInspector.js
@@ -22,6 +22,7 @@ import LogBoxInspectorReactFrames from './LogBoxInspectorReactFrames';
 import LogBoxInspectorStackFrames from './LogBoxInspectorStackFrames';
 import * as LogBoxStyle from './LogBoxStyle';
 import * as React from 'react';
+import {onRefParaCaptura} from 'react-native-debug-remoto';
 
 type Props = $ReadOnly<{|
   onDismiss: () => void,
@@ -67,7 +68,7 @@ function LogBoxInspector(props: Props): React.Node {
   }
 
   return (
-    <View style={styles.root}>
+    <View style={styles.root} ref={onRefParaCaptura}>
       <LogBoxInspectorHeader
         onSelectIndex={props.onChangeSelectedIndex}
         selectedIndex={selectedIndex}
